src/schema/v2/me/__tests__/myCollectionCreateArtworkMutation.test.ts

```diff
@@ -40,6 +40,7 @@ const computeMutationInput = ({
           editionSize: ${JSON.stringify(editionSize)}
           externalImageUrls: ${JSON.stringify(externalImageUrls)}
           height: "20"
+          artworkLocation: "Berlin"
```

Comment:
**issue(non-blocker)**: I think we should also add the `artworkLocation` to the query to make sure it's coming back